clients/venice-thin-client/src/main/java/com/linkedin/venice/client/store/StatTrackingStoreClient.java

```diff
@@ -252,21 +254,21 @@ public void compute(
   private static void handleMetricTrackingForStreamingCallback(
       ClientStats clientStats,
       long startTimeInNS,
-      Optional<VeniceClientException> veniceException,
+      Optional<Exception> exception,
```

Comment:
Can we just use nullable Exception instead of Optional ?

Reply:
`Optional` provides more safety and minimal overhead in this particular case, since Exceptions are rare and `Optional.empty()` is free